### PR TITLE
vlan: update error message pattern for new guest

### PR DESCRIPTION
**Pull request by sitoliu**
and replace "iptables -F" with stopping "firewalld.service"
ID: 1646793
Signed-off-by: Sitong Liu <siliu@redhat.com>

### CODE REVIEW

File: generic/tests/cfg/vlan.cfg [hunk 1]
```diff
@@ -13,7 +13,9 @@
     kill_vm_vm2 = yes
     kill_vm_gracefully_vm2 = no
     cmd_type = ip
-    msg_pattern = 'RTNETLINK answers: Numerical result out of range'
+    msg_pattern = '8021q: Invalid VLAN id'
+    RHEL.7:
```
Comment (vivianQizhu) [inline generic/tests/cfg/vlan.cfg]: Do we consider RHEL.6? Otherwise should have a `no RHEL.6` variant.

Comment (sitoliu) [reply generic/tests/cfg/vlan.cfg]: @vivianQizhu Thanks, will update it later.

Comment (sitoliu) [reply generic/tests/cfg/vlan.cfg]: @vivianQizhu Updated, please help look again, thanks.

**Comment by vivianQizhu** [comment]
@pezhang @xiywang Please help review, thanks.

**Comment by pezhang** [comment]
Looks good to me.

Acked by: Pei Zhang pezhang@redhat.com

**Comment by pezhang** [comment]
Test pass.

**Pull request merged**